net/crowdsec/files/crowdsec.defaults

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+CONFIG=/etc/crowdsec/config.yaml
+data_dir=`uci get "crowdsec.crowdsec.data_dir"`
+sed -i "s,^\(\s*data_dir\s*:\s*\).*\$,\1$data_dir," $CONFIG
+db_path=`uci get "crowdsec.crowdsec.db_path"`
+sed -i "s,^\(\s*db_path\s*:\s*\).*\$,\1$db_path," $CONFIG
+
+# Create data dir & permissions if needed
+if [ ! -d "${data_dir}" ]; then
+	mkdir -m 0755 -p "${data_dir}"
+fi;
+
+grep -q "login:" /etc/crowdsec/local_api_credentials.yaml
+if [ $? -eq 0 ]; then
```

Comment:
Technically this, and other instances like this, are correct (so no change is necessary), but I just wanted to let you know that since you are testing the exit value of a command (in this case, `grep`), you can run the command as part of the `if` statement directly:

```sh
if grep -q "login:" /etc/crowdsec/local_api_credentials.yaml; then
```

The `if` command tests if the exit value of given command is 0. `[` is an alias/symlink to the `test` command (and returns 0 if the given condition is true).

Reply:
will simplify this as recommended (not necessary but more readable).
thanks